.github/RELEASE.md

```diff
@@ -0,0 +1,265 @@
+[rever docs]: https://regro.github.io/rever-docs
+[compare]: compare
+[release notes]: releases/new
+
+### 1. Open an issue in the applicable project for the upcoming release with the following details.
+
+> **Note:**
+> The [epic template][epic template] is perfect for this, just remember to remove the https://github.com/conda/infra/labels/epic label.
+
+<details>
+<summary><code>GitHub Issue Template</code></summary>
+
+```markdown
+### Summary
+
+Placeholder for `conda 22.9.0` release.
+
+### Tasks
+
+[milestone]: https://github.com/conda/conda/milestone/56
+[releases]: https://github.com/conda/conda/releases
+[main]: https://github.com/AnacondaRecipes/conda-feedstock
+[conda-forge]: https://github.com/conda-forge/conda-feedstock
+
+- [ ] [Complete outstanding PRs][milestone]
+- [ ] Create release PR
+    - See release process https://github.com/conda/infra/issues/541
+- [ ] [Publish Release][releases]
+- [ ] Create/update `22.9.x` branch
+- [ ] Feedstocks
+    - [ ] Bump version [Anaconda's main][main]
+    - [ ] Bump version [conda-forge][conda-forge]
+    - Link any other feedstock PRs that are necessary
+- [ ] Handoff to the Anaconda packaging team
+- [ ] Announce release
+    - [ ] Slack
+    - [ ] Twitter
+```
+
+</details>
+
+
+### 2. Ensure `rever.xsh` and `news/TEMPLATE` is up to date.
+
+These are synced from https://github.com/conda/infra.
+
+<details>
+<summary><h3>3. Run Rever.</h3></summary>
+
+Currently, there are only 2 activities we use rever for, (1) aggregating the authors and (2) updating the changelog. Aggregating the authors can be an error-prone process and also suffers from builtin race conditions (i.e. to generate an updated `.authors.yml` we need an updated `.mailmap` but to have an updated `.mailmap` we need an updated `.authors.yml`). This is why the following steps are very heavy-handed (and potentially repetitive) in running rever commands, undoing commits, squashing/reordering commits, etc.
+
+1. Install [`rever`][rever docs] and activate the environment:
+
+    ```bash
+    $ conda create -n rever conda-forge::rever
+    $ conda activate rever
+    (rever) $
+    ```
+
+2. Clone and `cd` into the repository if you haven't done so already:
+
+    ```bash
+    (rever) $ git clone git@github.com:conda/conda.git
+    (rever) $ cd conda
+    ```
+
+2. Create a release branch:
+
+    ```bash
+    (rever) $ git checkout -b release-22.9.0
+    ```
+
+2. Run `rever --activities authors`:
+
+    > **Note:**
+    > Include `--force` when re-running any rever commands for the same `<VERSION>`, otherwise, rever will skip the activity and no changes will be made (i.e., rever remembers if an activity has been run for a given version).
+
+    ```bash
+    (rever) $ rever --activities authors <VERSION> [--force]
+    ```
+
+    - If rever finds that any of the authors are not correctly represented in `.authors.yml` it will produce an error, if the author that the error pertains to is:
```

Comment:
```suggestion
    - If rever finds that any of the authors are not correctly represented in `.authors.yml` it will produce an error. If the author that the error pertains to is:
```